### PR TITLE
Merge pull request #2 from dof1985/cwatm_wq

### PR DESCRIPTION
**Pull request by f-s-d**
Cwatm wq

### CODE REVIEW

File: cwatm/hydrological_modules/water_quality/erosed.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#
+# Created:     20/01/2022
+# Copyright:   (c) TT, FS, PB, MS, DF 2022
+# -------------------------------------------------------------------------
+
+
+from cwatm.management_modules.data_handling import *
+from cwatm.management_modules.globals import *
+
+class waterquality_erosed(object):
```
Comment (dof1985) [inline cwatm/hydrological_modules/water_quality/erosed.py:16]: What does 'erosed' stand for?

File: cwatm/hydrological_modules/water_quality/erosed.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        """
+    def __init__(self, model):
+        self.var = model.var
+        self.model = model
+
+    def initial(self):
+        """
+                INITIAL PART OF THE EROSED MODULE
+        """
+        print('erosed init')
```
Comment (dof1985) [inline cwatm/hydrological_modules/water_quality/erosed.py:47]: Choos one type on quotation marks for strings, either ' or "., see lines 47 and 53 for example

File: cwatm/hydrological_modules/water_quality/waterquality_main.py [hunk 3]
```diff
@@ -45,9 +46,10 @@ def __init__(self, model):
         
         # define water quality sub-modules
         self.waterquality_p = waterquality_phosphorus(self)
+        self.erosed = waterquality_erosed(model)
```
Comment (dof1985) [inline cwatm/hydrological_modules/water_quality/waterquality_main.py:49]: Here again about erosed - I would try to think on something a bit more intuitive to improve code readability. 
Maybe sediment?

File: cwatm/hydrological_modules/water_quality/waterquality_main.py [hunk 4]
```diff
@@ -57,27 +59,38 @@ def initial(self):
...
+                self.erosed.initial()
         
             
     def dynamic(self):
-        print("wq dyn.")
-    
-       
\ No newline at end of file
+
+        if self.var.includeWaterQuality:
```
Comment (dof1985) [inline cwatm/hydrological_modules/water_quality/waterquality_main.py:162]: It is not necessarily the place to run dynamic. I guess you might want to sync it with the soil module

**Review by dof1985 — CHANGES_REQUESTED** [review]
See comments in code.

Overall it looks good. There is just one main change I wish you implement - can you think of another name instead erosed? I find it is not intuitive enough, and would not be easily to interpret within the code. 

I am not sure about the change in gitignore - what is it about?

File: cwatm/hydrological_modules/water_quality/erosed.py (hunk 1 above)
Comment (f-s-d) [reply cwatm/hydrological_modules/water_quality/erosed.py:16]: EROsion SEDiment

**Review by dof1985 — APPROVED** [review]
You have handled the conflicts very well

**Pull request merged**